lnrpc/rpc.proto

```diff
@@ -24,868 +24,863 @@ option go_package = "github.com/lightningnetwork/lnd/lnrpc";
-    the index offset of the last entry. The index offset can be provided to the
-    request to allow the caller to skip a series of records.
-    */
-    rpc ForwardingHistory (ForwardingHistoryRequest)
-        returns (ForwardingHistoryResponse);
-
-    /* lncli: `exportchanbackup`
-    ExportChannelBackup attempts to return an encrypted static channel backup
-    for the target channel identified by it channel point. The backup is
-    encrypted with a key generated from the aezeed seed of the user. The
-    returned backup can either be restored using the RestoreChannelBackup
-    method once lnd is running, or via the InitWallet and UnlockWallet methods
-    from the WalletUnlocker service.
-    */
-    rpc ExportChannelBackup (ExportChannelBackupRequest)
-        returns (ChannelBackup);
-
-    /*
-    ExportAllChannelBackups returns static channel backups for all existing
-    channels known to lnd. A set of regular singular static channel backups for
-    each channel are returned. Additionally, a multi-channel backup is returned
-    as well, which contains a single encrypted blob containing the backups of
-    each channel.
-    */
-    rpc ExportAllChannelBackups (ChanBackupExportRequest)
-        returns (ChanBackupSnapshot);
-
-    /*
-    VerifyChanBackup allows a caller to verify the integrity of a channel backup
-    snapshot. This method will accept either a packed Single or a packed Multi.
-    Specifying both will result in an error.
-    */
-    rpc VerifyChanBackup (ChanBackupSnapshot)
-        returns (VerifyChanBackupResponse);
-
-    /* lncli: `restorechanbackup`
-    RestoreChannelBackups accepts a set of singular channel backups, or a
-    single encrypted multi-chan backup and attempts to recover any funds
-    remaining within the channel. If we are able to unpack the backup, then the
-    new channel will be shown under listchannels, as well as pending channels.
-    */
-    rpc RestoreChannelBackups (RestoreChanBackupRequest)
-        returns (RestoreBackupResponse);
-
-    /*
-    SubscribeChannelBackups allows a client to sub-subscribe to the most up to
-    date information concerning the state of all channel backups. Each time a
-    new channel is added, we return the new set of channels, along with a
-    multi-chan backup containing the backup info for all channels. Each time a
-    channel is closed, we send a new update, which contains new new chan back
-    ups, but the updated set of encrypted multi-chan backups with the closed
-    channel(s) removed.
-    */
-    rpc SubscribeChannelBackups (ChannelBackupSubscription)
-        returns (stream ChanBackupSnapshot);
-
-    /* lncli: `bakemacaroon`
-    BakeMacaroon allows the creation of a new macaroon with custom read and
-    write permissions. No first-party caveats are added since this can be done
-    offline.
-    */
-    rpc BakeMacaroon (BakeMacaroonRequest) returns (BakeMacaroonResponse);
+  /* lncli: `walletbalance`
```

Comment:
something went wrong on rebase here? Diff is huge

Reply:
Oh yes! My editor auto-formatted this file and changed the indentation. I have restored it.